opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp

```diff
@@ -316,9 +316,9 @@ void BlackoilAquiferModel<TypeTag>::createDynamicAquifers(const int episode_inde
         auto aquPos =
             std::find_if(std::begin(this->aquifers),
                          std::end(this->aquifers),
-                [id = id](const auto& aquPtr)
+                [Id = id](const auto& aquPtr)
```

Comment:
Same as in the other PR, why assign a local name in the first place?

Reply:
and answer is exactly the same, capturing structured bindings requires c++-20